pkg/storageops/aws/aws.go

```diff
@@ -43,6 +47,11 @@ func NewEnvClient() (storageops.Ops, error) {
 		return nil, ErrAWSEnvNotAvailable
 	}
 
+	instanceType := os.Getenv("AWS_INSTANCE_TYPE")
```

Comment:
Can you update the README.md for this new env variable?

Reply:
Also trim whitespaces here just to be safe since we are using this to detect nvme's later on.

Reply:
I think it should be done for all the variables here.  If one fails nothing from storageops/aws is going to really work. I will add it for all variables.

Reply:
Sounds good. Add it to `GetEnvValueStrict` in utils.go and use that instead.